web/book/src/syntax/expressions-and-operators.md

```diff
@@ -42,23 +41,18 @@ PRQL uses parentheses `()` for several purposes:
 - Parentheses group operands to control the order of evaluation, for example:
   `((1 + x) * y)`
 
-- Parentheses set off an [inner transform](./inner-transforms.md) for the
-  `group ()` and `window ()` transforms. _Question: Are there any other
-  transforms that use parentheses like this?_
+- Parentheses denote an [inner transform](./inner-transforms.md) for the
+  `group ()` and `window ()` transforms. 
 
-- Parentheses set off a minus sign for a function argument, for example:
+- Parentheses denote a minus sign for a function argument, for example:
```

Comment:
`denote` defn: "be a sign of; indicate."

I don't think that's quite right....

Reply:
Yeah, clunky and formal. But let's go with it.